lib/fhir_client/client.rb

```diff
@@ -208,6 +197,29 @@ def get_oauth2_metadata_from_conformance
       options
     end
 
+    def get_oauth2_metadata_from_service_defintion(rest)
```

Comment:
Sorry to be that guy who complains about spelling, but
`get_oauth2_metadata_from_service_defintion(rest)`
should be
`get_oauth2_metadata_from_service_definition(rest)`

Reply:
I use github as a distributed spell checker so.... Anyway, fixed.